### DISCUSSION TITLE
convert RTF text to HTML

### DISCUSSION THREAD

**Discussion by lokesh010** [adonisjs/core#1905] [Help]
I have inserted RTF text using `wysywig` editor. When I render it in `.edge` file it does not convert RTF to HTML. How do I format it?

**Comment by hariprasath-yadav** [comment]
I also want the solution, I am using WYSIWYG editor and save the data in HTML format in the database.
I want to render the HTML data from the database.
but, if i try to load the data from the database in edge template it displays as like HTML code
sample image is diplayed below
![Sample Image of output](https://user-images.githubusercontent.com/43471027/100537002-b45e9c00-324a-11eb-93f4-a4adfb03dce6.png)

**Comment by hariprasath-yadav** [comment]
Sorry i got the solution, insteed of using
`{{ data }}`
we have to use
`{{{ data }}}`
